postmaster/static/css/styles.css

```diff
@@ -289,36 +293,36 @@ a.adminLocked {
     }
 
     /* No More Tables */
-    #dynamicTable, 
-    #dynamicTable thead, 
-    #dynamicTable tbody, 
-    #dynamicTable th, 
-    #dynamicTable td, 
-    #dynamicTable tr { 
-        display: block; 
+    #dynamicTable,
```

Comment:
Can you make the rest of this file a separate commit called `Clean up whitespace`?